identity-engine/embedded-sign-in-widget/templates/home.gohtml

```diff
@@ -14,9 +14,9 @@
   <div>
     <p>Hello!</p>
     <p>If you're viewing this page then you have successfully configured and started this example server.</p>
-    <p>This example shows you how to to add the
-       <a href="https://developer.okta.com/authentication-guide/implementing-authentication/auth-code.html">Authorization Code Flow</a>
-       to your Golang application.</p>
+    <p>This example shows you how to 
```

Comment:
Embedded widget sample doesn't use the authorization code flow. It uses the interaction code flow.